### ISSUE TITLE
VoiceOver iOS: Moving focus automatically selects a radio

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-mcq#229]
### Subject of the issue
We have recently had an accessibility audit for one of our clients. The consulting agency flagged the following issue regarding the selection of options in an MCQ and GMCQ.

When navigating with VoiceOver on iOS, users cannot focus on a radio button without it also being selected. They said that "VoiceOver users will be unable to complete the test sections." However, I can still make a selection. It's just that I can't focus on an option without _also_ selecting it.

Their recommendation:

> Ensure that moving VoiceOver focus does not also automatically select a radio button. Focus should move freely between the radio buttons without automatically selecting one.

They also linked to a [Codepen](https://codepen.io/intopia/pen/zYqqgBR/30374b66c60d094700d031912d4e65b1) demonstrating how VoiceOver should interact with radio buttons. Clicking on an option does not select the radio button. Only a double-tap will do this.

Is this behavior intentional? Do we need to take a look at how this is implemented with regard to iOS VoiceOver?

**Type:** WCAG 2
**Principle:** Understandable
**Level:** A
**Severity:** Critical
**Guideline:** 3.2 Predictable
**Success Criteria:** 3.2.1 On Focus (A)

**Comment by oliverfoster** [comment]
If you look at the blame of these lines, where the behaviour deliberately originates:
https://github.com/adaptlearning/adapt-contrib-mcq/blame/13ca7d2be636598336660c6a2a90540b5262d51f/js/McqView.js#L29-L32

This commit https://github.com/adaptlearning/adapt-contrib-mcq/commit/e78b56787f80a6df90b9bc28f2ba80a12d546eb4

For this pr https://github.com/adaptlearning/adapt-contrib-mcq/pull/200
For this issue https://github.com/adaptlearning/adapt-contrib-mcq/issues/199

Having retested the previous issue, the issue now seems utterly counter intuitive and the solution is incorrect. 

I've made a pr to revert the behaviour and retested on ios, nvda and jaws. It now works fine and your issue above is fixed. There is a pr for both the mcq and gmcq, linked.
